Libraries/LibWeb/HTML/Canvas/CanvasState.h

```diff
@@ -11,6 +11,7 @@
 #include <AK/Vector.h>
 #include <LibGfx/AffineTransform.h>
 #include <LibGfx/Color.h>
+#include <LibGfx/Filter.h>
```

Comment:
"LibWeb/HTML: Scaffold canvas2d filters implementation" commit doesn't compile because `LibGfx/Filter.h` is introduced in subsequent commits. could you rearrange commit history so the project compiles on each individual commits? having commits that break build makes bisecting process very painful.